doc/classes/Control.xml

```diff
@@ -869,7 +869,7 @@
 			<argument index="0" name="control" type="Control">
 			</argument>
 			<description>
-				Shows the given control at the mouse pointer. A good time to call this method is in [method get_drag_data]. The control must not be in the scene tree.
+				Shows the given control at the mouse pointer. A good time to call this method is in [method get_drag_data]. The control must not be in the scene tree. You should not call [method Node.queue_free] on the control, and you should not keep a reference to the control beyond the duration of the drag. It will be deleted automatically after the drag has ended.
```

Comment:
I'd rather write "You should not free the control" or something similar. Besides `queue_free()` a user can use `free()` or `queue_delete()` (from SceneTree).